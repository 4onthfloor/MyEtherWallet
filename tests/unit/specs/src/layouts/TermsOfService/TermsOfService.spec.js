import { shallowMount } from '@vue/test-utils';
import TermsOfService from '@/layouts/TermsOfService/TermsOfService.vue';

import { Tooling } from '@@/helpers';

describe('TermsOfService.vue', () => {
  let localVue, i18n, wrapper, store;

  beforeAll(() => {
    const baseSetup = Tooling.createLocalVueInstance();
    localVue = baseSetup.localVue;
    i18n = baseSetup.i18n;
    store = baseSetup.store;
  });

  beforeEach(() => {
    wrapper = shallowMount(TermsOfService, {
      localVue,
      i18n,
      store,
      attachToDocument: true,
      mocks: {
        _t: () => {},
        $t: () => {}
      }
    });
  });

  it('should render correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('should dismount properly', () => {
    expect(wrapper.destroy()).toBe(undefined);
  });
});
