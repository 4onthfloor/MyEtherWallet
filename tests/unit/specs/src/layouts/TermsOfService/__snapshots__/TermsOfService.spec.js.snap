// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TermsOfService.vue should render correctly 1`] = `
<div>
  <div class="terms-and-conditions">
    <div class="wrap">
      <div class="page-container">
        <div class="content-width-limit">
          <div class="page-header">
            <h1></h1>
            <h6></h6> <br><br><br><br>
            <!---->
          </div>
          <div class="content">
            <h3>

            </h3> <br>
            <!----> <br>
            <p>

            </p> <br>
            <p>

            </p>
          </div> <br><br>
          <div class="content">
            <h3>

            </h3> <br>
            <p>

            </p> <br>
            <p>

            </p> <br>
            <p>

            </p> <br>
            <p>

            </p> <br>
            <p>

            </p> <br>
            <!---->
          </div> <br><br>
          <div class="content">
            <h3>

            </h3> <br>
            <p>

            </p> <br>
            <p>

            </p> <br>
            <p>

            </p> <br>
            <p>

            </p>
            <p>

            </p> <br>
            <p>

            </p>
            <p>

            </p> <br>
            <p>

            </p> <br>
            <p>

            </p> <br>
            <p>

            </p>
          </div> <br><br>
          <div class="content">
            <h3></h3> <br>
            <!---->
          </div> <br><br>
          <div class="content">
            <h3></h3> <br>
            <p>

            </p>
          </div> <br><br>
          <div class="content">
            <h3></h3> <br>
            <p>

            </p> <br>
            <p>

            </p> <br>
            <p>

            </p> <br>
            <p>

            </p> <br>
            <p>

            </p>
          </div> <br><br>
          <div class="content">
            <h3></h3> <br>
            <p>

            </p>
          </div> <br><br>
          <div class="content">
            <h3></h3> <br>
            <p>

            </p>
          </div> <br><br>
          <div class="content">
            <h3></h3> <br>
            <p>

            </p> <br>
            <p>

            </p> <br>
            <p>

            </p> <br>
            <p>

            </p> <br>
            <p>

            </p> <br>
            <p>

            </p> <br>
            <p>

            </p> <br>
            <p>

            </p> <br>
            <p>

            </p> <br>
            <p>

            </p> <br>
            <p>

            </p> <br>
            <p>

            </p> <br>
            <p>

            </p>
          </div> <br><br>
          <div class="content">
            <h3></h3> <br>
            <p>

            </p> <br>
            <p>

            </p>
          </div> <br><br>
          <div class="content">
            <h3></h3> <br>
            <p>

            </p>
          </div> <br><br>
          <div class="content">
            <h3></h3> <br>
            <!---->
          </div> <br>
        </div>
      </div>
    </div>
  </div>
</div>
`;
